helm/README.md

```diff
@@ -0,0 +1,34 @@
+# Metagrid Helm Chart
+
+## Install
+```shell
+helm install <name> oci://ghcr.io/esgf2-us/metagrid --version v1.3.2
```

Comment:
When I tried to run this step locally for testing, I got an error:
Error: INSTALLATION FAILED: YAML parse error on metagrid/templates/ingress.yaml: error converting YAML to JSON: yaml: line 14: did not find expected '-' indicator

Reply:
I also note you used --version  1.3.2 here, I'm guessing because that is the last tagged release? Maybe we can update this to 1.3.4 since I will be releasing a tag for 1.3.4 once the PR is ready

Reply:
If you do `helm install <name> oci://ghcr.io/esgf2-us/metagrid --version v1.3.2 --values test.yaml` it'll work. I'll need to adjust the defaults, I can either default to blank values or maybe have a `required` statement.

Reply:
Forgot to mention we'll need to re-upload that chart after it's fixed.